README.md

```diff
@@ -9,9 +9,9 @@
 
 This repo contains the source code for the distributed validator client *Charon*; a HTTP middleware client for Ethereum Staking that enables you to safely run a single validator across a group of independent nodes.
 
-Charon is accompanied by webapp called the [Distributed Validator Launchpad](https://github.com/obolnetwork/dv-launchpad), for distributed validator key creation.
+Charon is accompanied by a webapp called the [Distributed Validator Launchpad](https://github.com/obolnetwork/dv-launchpad) for distributed validator key creation.
```

Comment:
Since https://github.com/obolnetwork/dv-launchpad is not open-sourced yet, this hyperlink leads to nowhere

Reply:
Maybe rather remove that statement in that case

Reply:
@corverroos What about this one?